libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/viewer/MediaViewerView.kt

```diff
@@ -97,21 +110,37 @@ fun MediaViewerView(
             }
         )
         AnimatedVisibility(visible = showOverlay, enter = fadeIn(), exit = fadeOut()) {
-            MediaViewerTopBar(
-                actionsEnabled = state.downloadedMedia is AsyncData.Success,
-                mimeType = state.mediaInfo.mimeType,
-                onBackClick = onBackClick,
-                canDownload = state.canDownload,
-                canShare = state.canShare,
-                eventSink = state.eventSink
-            )
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .navigationBarsPadding()
+            ) {
+                MediaViewerTopBar(
+                    actionsEnabled = state.downloadedMedia is AsyncData.Success,
+                    senderName = state.mediaInfo.senderName,
+                    dateSent = state.mediaInfo.dateSent,
+                    onBackClick = onBackClick,
+                    eventSink = state.eventSink
+                )
+                MediaViewerBottomBar(
+                    modifier = Modifier.align(Alignment.BottomCenter),
+                    actionsEnabled = state.downloadedMedia is AsyncData.Success,
+                    canDownload = state.canDownload,
+                    canShare = state.canShare,
+                    mimeType = state.mediaInfo.mimeType,
+                    caption = state.mediaInfo.caption,
+                    onHeightChanged = { bottomPaddingInPixels = it },
```

Comment:
why do we need this padding computation?

Reply:
Yes